guestfs_generator/pyproject.toml

```
@@ -0,0 +1,59 @@
+[tool.poetry]
+name = "guestfs_generator"
+version = "0.0.0"
+description = ""
+authors = ["Matthew Willcockson <matthew@willcockson.family>"]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+dumb-pypi = "^1.5.1"
+beautifulsoup4 = "^4.9.3"
+orjson = "^3.4.7"
+importlib-metadata = {version = "^2.1.0", python = "<3.8"} # NOTE:FUTURE tox depends on <3.0.0
+lxml = "^4.6.2"
+pydantic = "^1.7.3"
+pylint = "^2.6.0"
+PySnooper = "^0.4.2"
+
+[tool.poetry.dev-dependencies]
+pytest = "^5.2"
+tox = "^3.21.2"
+pytest-cov = "^2.11.1"
+pytest-timeout = "^1.4.2"
+black = "^20.8b1"
+mypy = "^0.790"
+isort = "^5.6.4"
+
+[tool.pytest.ini_options]
+minversion = "6.2.0" # First version to have easily accessible types
+addopts = "--cov=guestfs_generator --strict-markers"
+# I don't plan on leaving parametersets empty
+# https://docs.pytest.org/en/stable/reference.html#confval-empty_parameter_set_mark
+empty_parameter_set_mark = "fail_at_collect"
+log_cli = true
+timeout = 60
+markers = [
+    "select: Temporary mark for selecting specific tests"
+]
+
+[tool.isort]
+profile = "black"
+# Ensures the output doesn't save if the resulting file contains syntax errors
+atomic = true
+# Add an explicitly defined source path (modules within src paths have their
+# imports automatically catorgorized as first_party)
+src_paths = ["guestfs_generator", "tests"]
+
+[tool.poetry.scripts]
+guestfs-generator = "guestfs_generator.main:main"
```

```suggestion
guestfs-generator = "guestfs_generator.main:main"
dumb-pypi = "dumb_pypi.main:main"
```

This should cause `poetry` to add a console script entrypoint to the distribution files.

This change can be added now without affecting anything.